### ISSUE TITLE
Vectorize design for calendar

### ISSUE THREAD

**Issue by aismail** [cami-project/cami-project#144]
## Why
We need a quick MVP to look at before starting to implement the calendar component in the client app.

## What
* [ ] a list of most used calendar components for React Native is built
* [ ] the following design is adapted to the capabilities of the existing components
![image uploaded from ios](https://cloud.githubusercontent.com/assets/82345/23548297/7f2807ea-000f-11e7-8a07-8cebf0afdf41.jpg)
* [ ] the adapted design is vectorized into digital format and delivered on #cami Slack channel

## Notes
The design must be in-line with what the existing libraries already offer! Lesson learned from the graphing components :)

**Comment by rtud** [comment]
After doing a fair bit of research, we have decided to base the Calendar screen on the [react-native-calendar](https://github.com/christopherdro/react-native-calendar) component:

- (+) as opposed to other components we've found, it is under active development
- (+) there are a lot of contributors and the component gathered a fair amount of stars on GH
- (-) the drawback is that the component doesn't currently have a Week view feature, so we'll be using the standard Month view for now

The vector design will take this component into consideration, thank you!

**Comment by rtud** [comment]
I have finished designing the high fidelity mockup for the Activities screen:

![CAMI Activities Screen v1](http://mq.d.pr/K18c.jpg)

Following a discussion with @asorici, we decided to go with a simpler & more UX friendly approach, centred around a vertical timeline, as apposed to a full fledged calendar screen (ex: iOS Calendar):

- (+) easier to browse through fewer events
- (+) induces the feeling of chronology
- (+) know at all times what the next event is
- (+) no longer requires an additional calendar table component
- (-) no historical view of events
- (-) harder to plan ahead

We're also taking into account that for the time being it will be much simpler for the Caregiver for example, to customize and plan using Google Calendar and have the events synced in the timeline fashion inside the app.

I will be creating a follow-up issue for implementing the design inside the application, and than move on to collecting and displaying actual data.

Closing, thank you!
/cc @aismail
